sbncode/CAFMaker/CAFMaker_module.cc

```diff
@@ -904,7 +923,23 @@ void CAFMaker::produce(art::Event& evt) noexcept {
     // }
 
     //#######################################################
-    // Add reconstructed objects.
+    // Add stub reconstructed objects.
+    //#######################################################
+    for (size_t iStub = 0; iStub < fmStubs.size(); iStub++) {
+      const sbn::Stub &thisStub = *fmStubs[iStub];
+      art::Ptr<recob::PFParticle> thisStubPFP;
+      if (fmStubPFPs.at(iStub).size()) thisStubPFP = fmStubPFPs.at(iStub).at(0);
+      rec.reco.nstub ++;
```

Comment:
I prefer the approach of setting `nstub` directly to the vector size used for the slice stubs below.

Reply:
Fixed.